### DISCUSSION TITLE
Rootless collabora container not working

### DISCUSSION THREAD

**Discussion by loeffelpan** [nextcloud/all-in-one#3725] [Questions] [answered]
I'm trying to get collabora working. But nextcloud is extremly slow when I activate "Collabora" in AIO webinterface.
Without collabora my setup works fine.

I followed #1358 and got to this output of `curl -vvv https://$NC_DOMAIN:443/hosting/discovery`:
```
*   Trying 93.220.206.86:443...
* Connected to kbw3.de (93.220.206.86) port 443
* ALPN: curl offers h2,http/1.1
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_CHACHA20_POLY1305_SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=kbw3.de
*  start date: Oct 29 18:11:16 2023 GMT
*  expire date: Jan 27 18:11:15 2024 GMT
*  subjectAltName: host "kbw3.de" matched cert's "kbw3.de"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://kbw3.de:443/hosting/discovery
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: kbw3.de]
* [HTTP/2] [1] [:path: /hosting/discovery]
* [HTTP/2] [1] [user-agent: curl/8.4.0]
* [HTTP/2] [1] [accept: */*]
> GET /hosting/discovery HTTP/2
> Host: kbw3.de
> User-Agent: curl/8.4.0
> Accept: */*
>
```
Then it stucks as there is no further response.

Maybe some addtional logs from the containers may help.
[apache.txt](https://github.com/nextcloud/all-in-one/files/13328526/apache.txt)
[collabora.txt](https://github.com/nextcloud/all-in-one/files/13328527/collabora.txt)
[nextcloud.txt](https://github.com/nextcloud/all-in-one/files/13328528/nextcloud.txt)

To be fair - I'm currently on a (unsupported) podman setup (#3487). 
I already disabled SELinux `enforced` mode to see if this is a security issue, but nothing changed.

Any help would be appreciated. Maybe someone can bring me on the right track.
Thanks in advice.

**Comment by szaimen** [comment, marked as answer]
Hi, I think you are most likely running into this: https://github.com/CollaboraOnline/online/issues/2800

**Reply by loeffelpan** [reply]
Thanks for linking me to the correct issue.
That means that running rootless (regardless of docker or podman) is not possible with collabora, correct?
Maybe that's something to mention in your rootless documentation.

**Reply by szaimen** [reply]
Right. Added with https://github.com/nextcloud/all-in-one/commit/2de67a38bdf0e2635679fafe12fb3dbdf917c344
